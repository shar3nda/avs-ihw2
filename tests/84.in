x1y{l>	j@E|p;xXcyX]aG<liG<aU;v>-R^*C(\eZ<B=<'=av]\E$J!CJOqQBmhN>56kBO`c[M~{^]X2;