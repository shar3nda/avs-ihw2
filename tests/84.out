Lowercase: 264, uppercase: 266